crates/chain/src/persist.rs

```diff
@@ -80,6 +80,15 @@ pub trait PersistBackend<C> {
 
     /// Return the aggregate changeset `C` from persistence.
     fn load_from_persistence(&mut self) -> Result<C, Self::LoadError>;
+
+    /// Returns whether the persistence backend contains no data.
+    fn is_empty(&mut self) -> Result<bool, Self::LoadError>
```

Comment:
why does this need to be on the trait. I don't see why we need it.

Reply:
This is for the `Wallet::new_or_load` method. So that we know whether we need to initialize (where we commit the first changeset) or load from existing. Technically, we can check whether required fields exists (i.e. network type and descriptors)... Would you prefer this way instead?